components/drop-zone/provider.js

```diff
@@ -92,21 +83,29 @@ class DropZoneProvider extends Component {
 		// Computing state
 		const { onVerifyValidTransfer } = this.props;
 		const isValidDrag = ! onVerifyValidTransfer || onVerifyValidTransfer( detail.dataTransfer );
+
+		// Always true (onVerifyValidTransfer is not defined).
```

Comment:
Can we remove it if it's always `true`?

Reply:
We can, although I can't tell what the original intention was to have this property in the first place. For the moment the property is not defined. Maybe @youknowriad can give us a bit more insight.

Reply:
I'm fine removing this, this is a leftover since this component has been adapted from Calypso